mass_mailing_sending_queue/models/mail_mass_mailing_sending.py

```diff
@@ -40,54 +47,51 @@ def _batch_size_get(self):
         return batch_size
 
     @api.multi
-    def pending_recipients(self):
+    def pending_emails(self):
+        self.ensure_one()
+        return self.env['mail.mail.statistics'].search([
+            ('mass_mailing_sending_id', '=', self.id),
```

Comment:
Not sure if what I'm going to say makes sense at all, but if you use `'in', self.ids`, you can remove the `self.ensure_one()`